solace-message-router/README.md

```diff
@@ -62,21 +69,24 @@ You need to fill in the following fields (* marks the mandatory ones):
 | **BASICS**                 |  |
 | Subscription*              | Provide your subscription to use. |
 | Resource Group*            | A new group, or an existing group that will be available from the pull-down menu once "Use existing" is selected. |
-| Location*                  | Specify a location for the resource group where metadata about the resources is stored. Select the region most suitable to you. |
```

Comment:
location is still a param in the template - is this removal intentional?

Reply:
Thanks for catching but intentional: This has removed a duplicate: `Location` is already listed further down with a default.